app/models/requests/mediated_page.rb

```diff
@@ -42,6 +42,8 @@ def item_limit
   end
 
   def requires_needed_date?
+    return false if origin == 'HOPKINS'
+    return false if origin_location == 'PAGE-MP'
```

Comment:
I think this is another indication of a sub-type that exists somewhere (similar to https://github.com/sul-dlss/sul-requests/blob/24bcfcae4cae13fbb7b5f77f6da410cf9fc82415/config/locales/en.yml#L88-L89)  :disappointed: